app/representers/api/v1/unclaimed_user_representer.rb

```diff
@@ -0,0 +1,13 @@
+module Api::V1
+  class UnclaimedUserRepresenter < Roar::Decorator
```

Comment:
My understanding was that all we should report back was the id for a user, so I didn't want to return the entire dataset.  You're correct though - it doesn't hurt to return a JSON blob that has the fields all empty.